games-arcade/oshu/oshu-2.0.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Lightweight osu! port"
+HOMEPAGE="https://github.com/fmang/oshu"
+SRC_URI="https://www.mg0.fr/oshu/releases/${P}.tar.gz
+	osu-skin? ( https://www.mg0.fr/oshu/skins/osu-v1.tar.gz -> ${PN}-skin-v1.tar.gz )"
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3 CC-BY-NC-4.0"
+SLOT="0"
+IUSE="libav osu-skin"
+
+CMAKE_MIN_VERSION="3.9.0"
+RDEPEND="
+	>=media-libs/libsdl2-2.0.5:=
+	media-libs/sdl2-image:=
+	x11-libs/cairo:=
+	x11-libs/pango:=
+	!libav? ( media-video/ffmpeg:= )
+	libav? ( media-video/libav:= )
+"
+
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
+"
+
+src_configure() {
+	local mycmakeargs=(
+		usex osu-skin \
```

Comment:
That is not what I meant, and it certainly does not work. Which sounds like you aren't testing what you write. E.g.:

    -DOSHU_DEFAULT_SKIN=$(usex osu-skin osu minimal)
    -DOSHU_SKINS=minimal$(usex usu-skin ;osu '')